app/entry.server.tsx

```diff
@@ -1,16 +1,22 @@
-import * as Sentry from "@sentry/remix";
+import * as Sentry from "@sentry/cloudflare";
```

Comment:
Great spot @geemanjs! 🙌 I'll raise an issue in the Sentry docs as their implementation mentions using the `@sentry/remix` import.